edge-functions/ab-testing-statsig/README.md

```diff
@@ -1,23 +1,23 @@
 ---
-name: A/B Testing with Statsig
+name: Experimentation with Statsig
 slug: ab-testing-statsig
-description: By A/B testing at the edge with Statsig, you'll reduce CLS and improve performance from client-loaded experiments while analyzing impact.
+description: By running experiments at the edge with Statsig, you'll reduce CLS and improve performance from client-loaded experiments while analyzing impact.
 framework: Next.js
 useCase:
   - Edge Functions
   - Documentation
 css: Tailwind
-deployUrl: https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/ab-testing-statsig&project-name=ab-testing-statsig&repository-name=ab-testing-statsig
-demoUrl: https://edge-functions-ab-testing-statsig.vercel.app
+deployUrl: https://vercel.com/new/clone?repository-url=https://github.com/vercel/examples/tree/main/edge-functions/ab-testing-statsig&project-name=ab-testing-statsig&repository-name=ab-testing-statsig&env=STATSIG_SERVER_API_KEY,NEXT_PUBLIC_STATSIG_CLIENT_KEY,STATSIG_CONSOLE_API_KEY,EDGE_CONFIG,EDGE_CONFIG_ITEM_KEY&envDescription=Statsig%20API%20keys%20and%20Edge%20Config%20settings&envLink=https%3A%2F%2Fdocs.statsig.com%2Fguides%2Ffirst-feature
+demoUrl: https://edge-functions-experimentation-statsig.vercel.app
```

Comment:
Is this change correct? based on the folder name I think it would be deployed to https://edge-ab-testing-statsig.vercel.app

Reply:
I don't have access to the project so I wasn't sure how the team wanted me to set this up. I'm hosting the demo on my own account at https://edge-function-edge-config-statsig.vercel.app/, happy to change this if the team wants to take over the hosting. I just noticed that the current live demo is outdated.

Reply:
we have an existing project for `edge-ab-testing-statsig` that gets deployed to https://edge-ab-testing-statsig.vercel.app - so this should work


this reminds me i guess we'd need to install the integration and fill it with the right env vars so the example works too

Reply:
cool i'll change the url back to the old one

Reply:
And yes you'll need to create a project that's setup properly, do you want me to create one for you in the Vercel account?

Reply:
updated the demo url back